packages/components/applicationTableComponent/PercentageVolumeCell.tsx

```diff
@@ -0,0 +1,48 @@
+import { StyleProp, View, ViewStyle } from "react-native";
+
+import downArrowSVG from "../../../assets/icons/downArrow.svg";
+import upArrowSVG from "../../../assets/icons/upArrow.svg";
+import { useIsMobile } from "../../hooks/useIsMobile";
+import { errorColor, successColor } from "../../utils/style/colors";
+import { fontSemibold11, fontSemibold13 } from "../../utils/style/fonts";
+import { layout } from "../../utils/style/layout";
+import { BrandText } from "../BrandText";
+import { SVG } from "../SVG";
+
+export const PercentageVolumeCell: React.FC<{
+  data: any;
+  style?: StyleProp<ViewStyle>;
+}> = ({ data, style }) => {
+  const isMobile = useIsMobile();
+  return (
+    <View
+      style={[
+        {
+          paddingRight: layout.spacing_x1,
+          flexDirection: "row",
+          flexWrap: "nowrap",
+          alignItems: "center",
+        },
+        style,
+      ]}
+    >
+      {data.includes("+") ? (
```

Comment:
Just a suggestion:
Since I see that data.includes("+") are used in > 1 place, then we can use a var for that. Ex
const isPositive = data.includes("+");

Then we can replace this code by shorter version
```<SVG source={isPositive ? upArrowSVG: downArrowSVG} />```